cli/command/formatter/disk_usage_test.go

```diff
@@ -23,6 +23,7 @@ func TestDiskUsageContextFormatWrite(t *testing.T) {
 Images              0                   0                   0B                  0B
 Containers          0                   0                   0B                  0B
 Local Volumes       0                   0                   0B                  0B
+Build Cache                                                 0B                  0B
```

Comment:
Wonder if we should have something (`-`) in the first two columns, but we can tweak that later